lms/djangoapps/grades/models.py

```diff
@@ -699,6 +700,7 @@ def update_or_create_override(
             subsection_grade_model: The PersistentSubsectionGrade object associated with this override.
             override_data: The parameters of score values used to create the override record.
         """
+        cls.changeReason = feature
```

Comment:
What is this?

Reply:
this is the code that allows us to add custom metadata into django simple history history_change_reason column. Right now it writes the source of grade change (GRADEBOOK) or PROCTORING

Reply:
Shouldn't it be on the instance, not the class?